connector-sdk/document/src/main/java/io/camunda/document/store/InMemoryDocumentStore.java

```diff
@@ -55,6 +55,10 @@ public String getContentType() {
 
           @Override
           public OffsetDateTime getExpiresAt() {
+            if (request.timeToLive() != null) {
+              return OffsetDateTime.now().plus(request.timeToLive());
+            }
+
             return null;
```

Comment:
Not sure if this returned always `null` on purpose or was just never implemented. Now it should behave as the API.